src/Mobs/AggressiveMonster.cpp

```diff
@@ -86,12 +85,13 @@ void cAggressiveMonster::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 
 bool cAggressiveMonster::Attack(std::chrono::milliseconds a_Dt)
 {
-	m_AttackInterval += (static_cast<float>(a_Dt.count()) / 1000) * m_AttackRate;
+
 	if ((m_Target == nullptr) || (m_AttackInterval < 3.0))
 	{
 		return false;
 	}
 
+
```

Comment:
Any particular reason for a double empty line? A single would have sufficed, in my opinion.